sources/packages/backend/apps/api/src/route-controllers/institution-locations/_tests_/e2e/institution-location.institutions.controller.getActiveApplications.e2e-spec.ts

```diff
@@ -0,0 +1,154 @@
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAuthRelatedEntities,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitutionLocation,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { ApplicationStatus, InstitutionLocation } from "@sims/sims-db";
+import { getUserFullName } from "../../../../utilities";
+
+describe("InstitutionLocationInstitutionsController(e2e)-getActiveApplications", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeFLocation: InstitutionLocation;
+  let collegeDLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({
+      institution: collegeF,
+    });
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+    // College D.
+    const { institution: collegeD } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeDUser,
+    );
+    collegeDLocation = createFakeInstitutionLocation({
+      institution: collegeD,
+    });
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeDUser,
+      collegeDLocation,
+    );
+  });
+
+  it("Should get the list of all applications which are not archived when requested for not archived application in Report a Change", async () => {
+    // Arrange
+    const completedApplication = await saveFakeApplication(
+      db.dataSource,
+      {
+        institutionLocation: collegeFLocation,
+      },
+      { applicationStatus: ApplicationStatus.Completed },
+    );
+
+    await saveFakeApplication(
+      db.dataSource,
+      {
+        institutionLocation: collegeFLocation,
+      },
+      { isArchived: true, applicationStatus: ApplicationStatus.Completed },
+    );
+
+    // Institution token.
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    const endpoint = `/institutions/location/${collegeFLocation.id}/active-applications?archived=false&page=0&pageLimit=10&sortField=applicationNumber&sortOrder=ASC`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        count: 1,
+        results: [
+          {
+            applicationId: completedApplication.id,
+            applicationNumber: completedApplication.applicationNumber,
+            applicationScholasticStandingStatus: "Available",
```

Comment:
Minor. Please use the enum from `import { ApplicationScholasticStandingStatus } from "../../../../services/application/application.models";`